docs/guides/expectations/creating_custom_expectations/how_to_create_custom_regex_based_column_map_expectations.md

```diff
@@ -0,0 +1,266 @@
+	* `out` is based on the Validation Result returned when executing the Expectation.
+	* `exact_match_out`: if you set `exact_match_out=False`, then you don’t need to include all the elements of the Validation Result object - only the ones that are important to test.
+
+
+If you run your Expectation file again, you won't see any new checkmarks, as the logic for your Custom Expectation hasn't been implemented yet. 
+However, you should see that the tests you've written are now being caught and reported in your checklist:
+
+```
+$ python expect_column_values_to_only_contain_vowels.py
+
+Completeness checklist for ExpectColumnValuesToOnlyContainVowels:
+  ✔ Has a library_metadata object
+  ✔ Has a docstring, including a one-line short description
+...
+	Has core logic that passes tests for all applicable Execution Engines and SQL dialects
+		  Only 0 / 2 tests for pandas are passing
+		  Only 0 / 2 tests for spark are passing
+		  Only 0 / 2 tests for sqlite are passing
+		  Failing: basic_positive_test, basic_negative_test
+...
+```
+
+:::note
+For more information on tests and example cases, <br/>
+see our guide on [how to create example cases for a Custom Expectation](../features_custom_expectations/how_to_add_example_cases_for_an_expectation.md).
+:::
+
+### 6. Define your regex and connect it to your Expectation
+
+This is the stage where you implement the actual business logic for your Expectation.   
+
+In the case of your Custom `RegexBasedColumnMapExpectation`, Great Expectations will handle the actual application of the regex to your data. 
+
+To do this, we replace these:
+
+```python file=../../../../examples/expectations/regex_based_column_map_expectation_template.py#L25-L27
+```
+
+with something like this:
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_values_to_only_contain_vowels.py#L17-L19
+```
+
+For more detail when rendering your Custom Expectation, you can optionally specify the plural form of a Semantic Type you're validating. 
+
+For example:
+
+```python file=../../../../examples/expectations/regex_based_column_map_expectation_template.py#L28
+```
+
+becomes:
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_values_to_only_contain_vowels.py#L20
+```
+
+Great Expectations will use these values to tell your Custom Expectation to apply your specified regex as a <TechnicalTag tag="metric" text="Metric"/> to be utilized in validating your data.
+
+This is all that you need to define for now. The `ColumnMapRegexExpectation` class has built-in logic to handle all the machinery of data validation, including standard parameters like `mostly`, generation of Validation Results, etc.
+
+<details>
+  <summary>Other parameters</summary>
```

Comment:
Does using this `ColumnMapRegexExpectation` class allow for specifying your own Success Keys? If so, you may want to mention this or have a throwaway example here (e.g. you could have an expectation around phone numbers which lets you pass in a flag to include or exclude parentheses and hyphens).